README.md

```diff
@@ -61,6 +61,39 @@ sw  $t3, 8($t0)    # the_list[i+1] = $t3
     * .ascii "Hello" does not null terminate (when string ends is not specified), so please use .asciiz
     * You cannot use ALU operations (arithmetic, logic) on memory, you must load them into registers first, then store the results
     * When doing comparisons, you usually want the exact opposite e.g. say you want if (x > 2) {...} you want to jump after the block on condition !(2 < x) (slt then bne)
+        * Example
```

Comment:
Markdown formatting is ruined :'(
Look at "view file" (probably under a ... dropdown), you want to reorder things so that there are no code blocks within nested dot points.